Packs/GitHub/Integrations/GitHub_IAM/GitHub_IAM.py

```diff
@@ -362,20 +362,20 @@ def main():
 
         elif command == 'iam-get-user':
             user_profile = get_user_command(client, args, mapper_in)
-            return_results(user_profile)
+            user_profile.return_outputs()
```

Comment:
revert these changes. keep using `return_results`
we should be consistent with how we return stuff from integration/scripts

Reply:
See my comment above.

Reply:
So we can change this to be:
`return_results(user_profile.to_entry())' if you prefer